app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -826,15 +826,17 @@ def test_reject_draft_experiment(self):
     def test_reject_ending_experiment(self):
         user_email = "user@example.com"
         experiment = NimbusExperimentFactory.create_with_lifecycle(
-            NimbusExperimentFactory.Lifecycles.ENDING_REVIEW_REQUESTED
+            NimbusExperimentFactory.Lifecycles.ENDING_REVIEW_REQUESTED,
+            is_rollout=False,
         )
         response = self.query(
             UPDATE_EXPERIMENT_MUTATION,
             variables={
                 "input": {
                     "id": experiment.id,
                     "publishStatus": NimbusExperiment.PublishStatus.IDLE.name,
-                    "statusNext": NimbusExperiment.Status.COMPLETE.name,
+                    "statusNext": None,
```

Comment:
why we are changing` statusNext `here?

Reply:
The test is supposed to be looking at a request to end the experiment being rejected, which means it's statusNext should be None. I'm not sure why it was using `COMPLETE` before, I think it was just an overlook 🤷‍♀️

Reply:
create an experiment with: 

```
status: LIVE
statusNext: COMPLETE
publishStatus: REVIEW
```

reject -> 

```
status: LIVE
statusNext: None
publishStatus: IDLE
```

Reply:
https://github.com/mozilla/experimenter/blob/main/app/experimenter/docs/README.md#end-reject------

Reply:
A+ for referring back to the diagrams here 👏 👏 👏 👏